include/bl32/sp_min/platform_sp_min.h

```diff
@@ -17,4 +17,6 @@ void sp_min_plat_runtime_setup(void);
 void sp_min_plat_arch_setup(void);
 entry_point_info_t *sp_min_plat_get_bl33_ep_info(void);
 
+void plat_sp_min_interrupt_handler(uint32_t id);
```

Comment:
The other functions in this file are called `sp_min_plat_xxx()`. You should align with this. Also wouldn't `sp_min_plat_fiq_handler()` be more accurate?